blockmanager_test.go

```diff
@@ -440,7 +440,7 @@ func TestBlockManagerInitialInterval(t *testing.T) {
 		if test.partialInterval {
 			startHeight = wire.CFCheckptInterval / 3
 		}
-		go func() {
+		go func() { // nolint:staticcheck
```

Comment:
Can we remove the lint exception for all the `t.Fatal` in goroutines and follow https://golang.org/doc/go1.16#vet?

Reply:
Ah, I didn't know it was that easy to fix (expected a bigger refactor). Thanks for the hint, done!